app/models/repository/git.rb

```diff
@@ -1,6 +1,6 @@
 require 'git_repository'
 
-module Repository::GitRepositories
+module Repository::Git
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.